generators/client/templates/vue/src/main/webapp/app/router/index.ts.ejs

```diff
@@ -35,6 +35,12 @@ const <%= jhiPrefixCapitalized %>TrackerComponent = () => import('../admin/track
 // jhipster-needle-add-entity-to-router-import - JHipster will import entities to the router here
 
 Vue.use(Router);
+
+const router = new Router({
```

Comment:
Does is create second instance of the `router` as the export also does `new Router`

Reply:
Just tested without this part and it works well !
Good job @sabrinapayet, it looks like it was more complex than expected

Reply:
I push directly to your branch, to fix this